crypto/x509/x509_test.cc

```diff
@@ -1452,6 +1477,17 @@ TEST(X509Test, TestPSS) {
   ASSERT_TRUE(X509_verify(cert.get(), pkey.get()));
 }
 
+TEST(X509Test, TestRsaSsaPss) {
```

Comment:
Again, we should also test the unhappy path and think about some good vectors we want to test. We want to raise the unit testing bar.

Reply:
No unhappy path because no new logic (except identifying new OID).

Reply:
That sounds like new logic to me? if there are few failure cases, it should be fast to add a test that hits it.

Reply:
The new logic of this PR only includes
* rsassapss OID -> ameth mapping. so loading key can find the corresponding key.
* removed the null param restriction, which is not applied to rsassapss based on rfc.